LINTS.md

```diff
@@ -0,0 +1,68 @@
+PL/Rust uses its own "rustc driver" so that it can employ custom lints to detect certain Rust code idioms and patterns
+that trigger "I-Unsound" bugs in Rust itself.  Think "clippy" but built into the Rust compiler itself. In addition to 
+these custom lints, PL/Rust uses some standard Rust lints to enforce safety.
+
+In all cases, these lints are added to the generated code which wraps the user's `LANGUAGE plrust` function, as 
+`#![forbid(${lint_name})]`.  They are used with "forbid" to ensure a user function cannot change it back to "allow".
+
+PL/Rust does **not** apply these lints to dependant, external crates.  Dependencies *are* allowed to internally use 
+whatever code they want, including `unsafe`.  Note that any public-facing `unsafe` functions won't be callable by a plrust 
+function.
+
+Dependencies are granted more freedom as the usable set can be controlled via the `plrust.allowed_dependencies` GUC.
+
+> **It is the administrator's responsibility to properly vet external dependencies for safety issues that may impact
```

Comment:
Hm. The `>` indicates a quotation? And also causes this to look de-emphasized on my display.

```suggestion
**It is the administrator's responsibility to properly vet external dependencies for safety issues that may impact
```